lib/blockchainexplorer.js

```diff
@@ -14,19 +14,13 @@ function BlockChainExplorer(opts) {
   $.checkArgument(opts);
   var provider = opts.provider || 'insight';
   var network = opts.network || 'livenet';
+  var dfltUrl = network == 'livenet' ? 'https://insight.bitpay.com:443' :
```

Comment:
It is on the conf file, I just wanted to give a nice dflt value if the keys are not present of the config file.